packages/docs/celo-holder-guide/ledger.md

```diff
@@ -169,3 +153,18 @@ If you have issues connecting to the Ledger, try the following:
 - Ensure that you are running the latest version of the Celo CLI.
 
 There have been reports of a possible [issue](https://github.com/celo-org/celo-ledger-spender-app/issues/13) that appears to affect developer store apps on the Ledger Nano X including the Celo Ledger App. This is believed to be fixed in version 1.0.3. In earlier versions, a user clicking through the `Pending Ledger review` notice too rapidly can cause the device to freeze. If this occurs, wait until the device's battery is depleted, then charge and power up again. Then use Ledger Live Manager to update the installed version of the Celo Ledger App.
+
+## Using Your Ledger with Celo Wallet app
+
+An alternative to using `celocli` is [celowallet.app](https://celowallet.app)
```

Comment:
A little more context may be helpful here. Maybe we can add something like:
`which supports Ledger and provides a simple interface for transacting with Celo accounts`.